.github/workflows/propose-release.yml

```diff
@@ -37,4 +37,4 @@ jobs:
 
         sed -i 's#https://flakehub.com/f/DeterminateSystems/nix/=.*";#https://flakehub.com/f/DeterminateSystems/nix/=${{ inputs.nix-version }}";#' ./flake.nix
         git add flake.nix
-        git commit -m "Update Nix release to ${{ inputs.nix-version }}"
+        git commit -m "Update Nix release to ${{ inputs.nix-version }}" || true
```

Comment:
Do we want this or `--allow-empty-commit`?

Reply:
I think ||true, since it isn't actually updating the Nix release if there is no diff.

Reply:
To clarify: I'm not asking because I care either way, I just want to document our alternative as being either "it's okay to skip" or "there should always be a commit".

Reply:
Yeah, cool. It's okay to skip.